recipe/meta.yaml

```diff
@@ -64,7 +64,7 @@ requirements:
     - libarrow =16
     - libboost-headers
     - libparquet =16
-    - libprotobuf
+    - libprotobuf =4.25.3
```

Comment:
This is the currently pinned version: https://github.com/conda-forge/csp-feedstock/blob/834f80f49e544c611079730df0e6a37449344c2e/.ci_support/linux_64_numpy1.22python3.10.____cpython.yaml#L22 Don't add a pin here as this stop future migrations. Just don't merge migration PRs for not.

Reply:
I am missing some context here. If I don't pin the libprotobuf version here to either `<5` or specific version  `4.*.*` the builds all fail.

Reply:
No, they shouldn't fail as this will take the global pin which is at the 4.25.t.

Reply:
Right, if I remove this global pin it won't take the other versions.